doc/src/challenges/0011-floats-ints.md

```diff
@@ -0,0 +1,55 @@
+# Challenge 11: Safety of Floats and Integers in `core::num`
+
+
+- **Status:** Open
+- **Tracking Issue:** [Link to issue](https://github.com/model-checking/verify-rust-std/issues/59)
+- **Start date:** *2024-08-20*
+- **End date:** *2024-12-10*
+
+-------------------
+
+## Goal
+
+Verify the safety of public unsafe methods for floats and integers in `core::num`.
+
+## Success Criteria
+
+### Signed Integers
+
+Prove the absence of undefined behavior in the following methods:
+
+1. `int_macros::unchecked_add`
+2. `int_macros::unchecked_sub`
+3. `int_macros::unchecked_mul`
+4. `int_macros::unchecked_neg`
+5. `int_macros::unchecked_shl`
+6. `int_macros::unchecked_shr`
+
+for each of the [signed integer types](https://doc.rust-lang.org/beta/book/ch03-02-data-types.html#integer-types).
+
+### Unsigned Integers
+
+Prove the absence of undefined behavior in the following methods:
+
+1. `uint_macros::unchecked_add`
+2. `uint_macros::unchecked_sub`
+3. `uint_macros::unchecked_mul`
+4. `uint_macros::unchecked_shl`
+5. `uint_macros::unchecked_shr`
+
+for each of the [unsigned integer types](https://doc.rust-lang.org/beta/book/ch03-02-data-types.html#integer-types). 
+
+### Float to Integer Conversion
+
+Verify the absence of undefined behavior in `to_int_unchecked` for `f16`, `f32`, `f64`, and `f128`. 
+
+## List of UBs
```

Comment:
Can you add arithmetic overflow?

Reply:
To the list of UBs? I thought that in safe and unsafe Rust, arithmetic overflow is defined (panic in debug mode, wrap in twos complement). Assuming that your code doesn't overflow could cause later undefined behavior (e.g., accessing out of bounds memory) but the overflow itself is defined.

Reply:
I think we can add to the success criteria to report any possible instance of arithmetic overflows. We don't need to add to the list of UBs, because it's not.